tpu-client/src/tpu_client.rs

```diff
@@ -95,6 +105,66 @@ where
         self.invoke(self.tpu_client.try_send_transaction(transaction))
     }
 
+    /// Serialize and send transaction to the current and upcoming leader TPUs according to fanout
```

Comment:
I would use `.` and `s` for 3rd person verbs endings everywhere or nowhere (not sure how we do in another places):

```rust
/// Serialize and send transaction to the current and upcoming leader TPUs according to fanout
/// size.
/// Attempt to send and confirm tx `tries` times.
/// Wait for signature confirmation before returning.
/// Return the transaction signature.
```